internal/handlers/helpers_test.go

```diff
@@ -0,0 +1,38 @@
+package handlers
+
+import (
+	"fmt"
+	"github.com/go-chi/chi/v5"
+	"github.com/redhatinsights/mbop/internal/config"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+// TestSendJSONWithStatusCodeContentTypeHeader tests that the helper function returns an "application/json" value for
+// the "Content-Type" header.
+func TestSendJSONWithStatusCodeContentTypeHeader(t *testing.T) {
+	config.Get().UsersModule = "mock"
+
+	testRouter := chi.NewRouter()
+	testRouter.Get("/v3/accounts/{orgID}/users", AccountsV3UsersHandler)
+
+	testServer := httptest.NewServer(testRouter)
+	defer testServer.Close()
+
+	fullURL := fmt.Sprintf("%s/v3/accounts/12345/users", testServer.URL)
+	response, err := http.Get(fullURL) // nolint because the test server's URL is dynamic.
```

Comment:
I have ignored the linter's error about "http request made with variable url" because it's in the tests and we know where these requests are headed to. However if there's a better way of dealing with this I am all ears :smile: